### ISSUE TITLE
weave launch not working on Fed23

### ISSUE THREAD

**Issue by setiseta** [weaveworks/weave#1667]
If i run weave launch on Fedora 23 i get this error:
2015/11/11 10:21:27 Post http:///var/run/docker.sock/v1.15/containers/create: EOF

I don't know if its related to weave or to fedora.
Maybe someone can help.

**Comment by dpw** [comment]
It's a fedora thing.  I saw this when it was in updates-testing, I wasn't expecting them to promote it to updates without fixing the issue.  I'll dig out the details...

**Comment by dpw** [comment]
Fedora bug: https://bugzilla.redhat.com/show_bug.cgi?id=1275593

So it works for me if I remove the update:

```
# systemctl stop docker
# dnf remove docker
# dnf --disablerepo=updates install docker
# systemctl start docker
```

(That gives you docker 1.7.0, and it will break again the next time you do `dnf upate`, so it's not a real solution.)

Closing this, as there's not much that can be done except to wait for fedora to fix it.

**Comment by dpw** [comment]
Seems to be resolved by docker-1.8.2-14.git8f9eabc.fc23

**Comment by setiseta** [comment]
yes its working now. thx for reply
